src/index.ts

```diff
@@ -162,5 +170,52 @@ export function transposeTarStream(tarStream: NodeJS.ReadableStream,
 
 		tarStream.pipe(extract)
 	})
+}
+
+/**
+ * getBuildThroughStream: Get a through stream, which when piped to will remove all
+ * extra output that is added as a result of this module transposing a Dockerfile.
+ *
+ * This function enables 'silent' emulated builds, with the only difference in output
+ * from a native build being that there is an extra COPY step, where the emulator is
+ * added to the container
+ */
+export function getBuildThroughStream(opts: TransposeOptions): NodeJS.ReadWriteStream {
+
+	// Regex to match against 'Step 1/5:', 'Step 1/5 :' 'Step 1:' 'Step 1 :'
+	// and all lower case versions.
+	const stepLineRegex = /^(?:step)\s\d(?:\/\d)?\s?:/i
```

Comment:
This only supports single digit numbers, so it won't match `Step 10:` or `Step 1/10:` for example, which will break this.